app/scenes/Document/components/SharePopover.tsx

```diff
@@ -233,6 +247,15 @@ function SharePopover({
               </SwitchText>
             </SwitchLabel>
           </SwitchWrapper>
+          <Separator />
+          <SwitchWrapper>
+            <Input
+              type="text"
+              label={t("Share url slug")}
```

Comment:
```suggestion
              label={t("Custom link")}
```

Perhaps?

Reply:
Sure. It goes well with the actual link preview just below the input to resolve a minor confusion arising out of "Custom link" - the input field expects a slug, not a link as the label suggests :smile: